lua/visual.lua

```diff
@@ -183,8 +183,11 @@ core.register_entity("drawers:visual", {
 					inventoryChanged = true
 				end
 
-				-- set new stack
-				inv:set_stack("main", i, leftover)
+				-- set new stack if not creative
+				if not (minetest.check_player_privs(clicker, "creative") or
+					minetest.is_creative_enabled("")) then
```

Comment:
```Lua
local player_name = clicker:get_player_name()
```
^ outside of the loop

```suggestion
				-- set new stack if not creative
				if not minetest.is_creative_enabled(player_name) then
```

`creative`-like mods must override this API function to take their privileges into account. Same idea as with `minetest.is_protected`.

Requires Minetest 5.3.0 or newer.

Reply:
ohhh, ok. I misunderstood the description for that function. TYVM

Reply:
Done

Reply:
Oh. I didn't mean to imply that you should replace it in the entire mod. Anyway... this is fine I guess.